python/README.rst

```diff
@@ -36,14 +53,13 @@ Install python3.7 and use it for this directory
 .. code-block:: bash
```

Comment:
I think L34 should read something along the lines of, "Set the local Python version for our project to 3.7.4"